tests/handlers/test_query_results.py

```diff
@@ -29,6 +29,16 @@ def test_returns_404_if_no_cached_result_found(self):
         self.assertEqual(404, rv.status_code)
 
 
+class TestQueryResultsContentDispositionHeaders(BaseTestCase):
+    def test_supports_unicode(self):
+        query_result = self.factory.create_query_result()
+        query = self.factory.create_query(name="עברית", latest_query_data=query_result)
+
+        rv = self.make_request("get", "/api/queries/{}/results.json".format(query.id))
+        # This is what gunicorn will do with it
+        rv.headers['Content-Disposition'].encode('ascii')
```

Comment:
Is this an implicit assertion?

Reply:
Yes. I don't care to test the content, sort of assume it's fine. I just want to make sure it doesn't raise the exception.

Reply:
I see, in that case - how about:

```suggestion
        try:
            rv.headers['Content-Disposition'].encode('ascii')
        except Exception as e:
            self.fail(repr(e))            
```